msg/MotionPlanRequest.msg

```diff
@@ -51,8 +51,8 @@ float64 allowed_planning_time
 float64 max_velocity_scaling_factor
 float64 max_acceleration_scaling_factor
 
-# Maximum cartesian speed for the given end effector.
+# Maximum cartesian speed for the given link.
 # If max_cartesian_speed <= 0 the trajectory is not modified.
 # These fields require the following planning request adapter: default_planner_request_adapters/SetMaxCartesianEndEffectorSpeed
-string cartesian_speed_end_effector_link
+string cartesian_speed_link
```

Comment:
```suggestion
string cartesian_speed_limited_link
```
While we're renaming, why not make it a little easier to understand?

Reply:
Good point, thanks